docs/source/user-guide/user_interface/integrations.rst

```diff
@@ -0,0 +1,122 @@
+=============
+Integrations
+=============
+
+
+Notifications
+=============
+
+
+Email Notifications
+-------------------
+
+To configure email notifications go to the page 'Configurations/Integrations' and click on the tab 'Notifications'.
```

Comment:
@yromanyshyn swap tab 'Notifications' -> to  'Notifications' tab

Reply:
fixed